tests/core/test_overdrive.py

```diff
@@ -552,8 +555,21 @@ class MockAPI:
 
         extractor = OverdriveRepresentationExtractor(MockAPI())
         advantage_data = extractor.book_info_to_circulation(info)
-        assert None == advantage_data.licenses_owned
-        assert None == advantage_data.licenses_available
+        assert 0 == advantage_data.licenses_owned
+        assert 0 == advantage_data.licenses_available
+        assert 0 == consortial_data.patrons_in_hold_queue
```

Comment:
`consortial_data` hasn't been updated since l. 524 and was already tested on l. 542, so probably no need to test it again here.

Also, again because the test response file being used is associated with only one of the these API calls, not sure ow the "numberOfHolds" value could be reasonable applied to any other OD library.

Same below on l. 573.